### DISCUSSION TITLE
Homebrew Formulas Dictionary

### DISCUSSION THREAD

**Discussion by kevintraver** [streetsidesoftware/cspell-dicts#2562] [Ideas]
Idea: New dictionary for all **_homebrew_** formulas

**Comment by Ki-er** [comment]
I can't seem to find a definitive text based list of all formulae, I found [this](https://formulae.brew.sh/formula/) but it would require a lot of copy and pasting, which I do not mind doing when I have some time.

**Comment by Ki-er** [comment]
I have created a homebrew dict - see #2563

**Reply by Jason3S** [reply]
Hello Kieran,

Thank you for creating a dictionary.

Just so I understand, what is the use case for this dictionary? When would someone need it?

**Reply by Ki-er** [reply]
I think @kevintraver would be better to ask as the requester for this dict however a couple ideas come to mind

1. A list of packages installed for easy reference in the future, e.g a fresh install 
2. A documentation site that details packages and what they do

**Comment by Jason3S** [comment]
@kevintraver,

I'm not seeing a big need to maintain a dictionary of Homebrew Formulas here. If you need to spell check a document that contains the formulas, then a `brew list -1 > formulas.txt` will get you the list stored on your MacOS. This file can then be used as a dictionary in `cspell`.

My concern about maintaining a comprehensive list is that the list of Homebrew Formulas is very dynamic and will need an automated way to keep it updated. Now this is possible, but I would want to make sure that it serves a purpose. Most of the tools listed in Homebrew will also be listed under software-tools or other dictionaries.

**Reply by kevintraver** [reply]
Ok that makes total sense.
